src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -14,4 +21,20 @@
 public class WalletController {
 
     private final WalletService walletService;
+
+    @GetMapping(value = "/wallets")
```

Comment:
Niepotrzebne to value, nad klasą adnotacja @RequestMapping to załatwi, jak będzie więcej metod to po co w każdej znowu value dodawać

Reply:
Done, name of the attribute has been removed from annotation, only the passed  value remained. Thanks.